### PR TITLE
feat(helm): add deploymentLabels variable

### PR DESCRIPTION
**Pull request by mathieuherbert**
Signed-off-by: Mathieu Herbert <math.herbert@gmail.com>

add deploymentLabels as we have for annotations.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: helm/superset/values.yaml [hunk 1]
```diff
@@ -278,8 +278,11 @@ supersetNode:
         - /bin/sh
         - -c
         - dockerize -wait "tcp://$DB_HOST:$DB_PORT" -timeout 120s
+
   # -- Annotations to be added to supersetNode deployment
   deploymentAnnotations: {}
+  # -- Labels to be added to supersetNode deployment
+  deploymentLabels: {}
```
Comment (craig-rueda) [inline helm/superset/values.yaml:285]: Pls update the schema to reflect this addition

Comment (mathieuherbert) [reply helm/superset/values.yaml:285]: Hi, fixed thanks !

**Comment by craig-rueda** [comment]
Still looking like there's failed checks

**Comment by craig-rueda** [comment]
Pls rebase after this merges: https://github.com/apache/superset/pull/22590

(should fix linting errors)

**Review by craig-rueda — APPROVED** [review]

**Comment by craig-rueda** [comment]
Pls make sure you run helm-docs to generate the readme. Looks like that check is still failing

**Pull request merged**